ForgeRock Cloud APIs.postman_collection.json

```diff
@@ -659,6 +675,260 @@
 						}
 					]
 				},
+				{
+					"name": "Get Refresh Token",
```

Comment:
Since this GET refresh token call creates new values for access_token, refresh_token, and id_token, I copy the same script here